internal/scan-manager/common/src/main/java/org/wso2/security/tools/scanmanager/common/ScanLogRequest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.security.tools.scanmanager.common;
+
+/**
+ * Model class to represent a scan log request object coming from scanner micro services.
```

Comment:
```suggestion
 * Represents a scan log request coming from scanner micro services.
```

Reply:
- Needless to say that this is a "model" 
- Needless to say this creates an object.